modules/dxp/apps/osb/osb-faro/osb-faro-service/src/main/java/com/liferay/osb/faro/internal/upgrade/v17_0_0/UpgradeFaroProjectEmailDomain.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+
+package com.liferay.osb.faro.internal.upgrade.v17_0_0;
+
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+import com.liferay.portal.kernel.upgrade.UpgradeProcessFactory;
+import com.liferay.portal.kernel.upgrade.UpgradeStep;
+import com.liferay.portal.kernel.util.StringBundler;
+
+/**
+ * @author Leilany Ulisses
+ */
+public class UpgradeFaroProjectEmailDomain extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		alterTableName(
+			"OSBFaro_FaroProjectEmailAddressDomain",
+			"OSBFaro_FaroProjectEmailDomain");
+
+		runSQL(
+			StringBundler.concat(
+				"alter table OSBFaro_FaroProjectEmailAddressDomain rename to ",
+				"OSBFaro_FaroProjectEmailDomain and alter column ",
+				"faroProjectEmailAddressDomainId rename to ",
+				"faroProjectEmailDomainId"));
```

Comment:
Do we really need to have calling both `alterTableName` and `runSQL`?
Please, see an example of usage [here](https://github.com/liferay/liferay-portal/blob/1c22ad85dadc842c7474919d89325ca7d384dc3d/modules/apps/client-extension/client-extension-service/src/main/java/com/liferay/client/extension/internal/upgrade/v3_0_0/ClientExtensionEntryUpgradeProcess.java#L31)